src/action/mod.rs

```diff
@@ -414,6 +414,8 @@ pub enum ActionError {
     /// A MacOS (Darwin) plist related error
     #[error(transparent)]
     Plist(#[from] plist::Error),
+    #[error("Unexpected binary tarball contents found, the build result of `nix build nix#hydraJobs.binaryTarball.$SYSTEM` is expected")]
```

Comment:
Not a blocker, but I don't know if we should tell them about the `nix build` command; maybe point them to https://nixos.org/manual/nix/stable/installation/installing-binary.html#installing-from-a-binary-tarball instead?

Reply:
Hmmm that section doesn't seem too helpful...

Reply:
Ideally I'd prefer to point them at a page like https://github.com/NixOS/nix/tags where they could download the binary tarballs, but I'm not aware of one... https://releases.nixos.org/nix/ is just a 404.

Reply:
IMHO, neither is telling them to run a `nix` command when the tool supposed to be installing Nix failed :laughing:

Reply:
I mean, fair! How about we just skip the recommendation.

Reply:
https://releases.nixos.org/?prefix=nix/ Will give you all the nix releases (it's the S3 HTML frontend whatever thing)

Reply:
But yes, I'm also in favor of dropping the recommendation altogether.

Reply:
I put in the url!